ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2343,3 +2343,51 @@ periodics:
     - name: github-app-credentials
       secret:
         secretName: openshift-prow-github-app
+- agent: kubernetes
+  cluster: app.ci
+  cron: 0 */6 * * 1-5
```

Comment:
Could we do it every day? If someone merges a new job on Friday, we'd miss 2 days of data since I don't think we'd collect any data until Monday morning.

Reply:
Nevermind, I misread the cron, this is fine!